src/main/java/com/onarandombox/MultiverseCore/utils/WorldManager.java

```diff
@@ -193,16 +193,31 @@ public boolean cloneWorld(String oldName, String newName) {
 
         if (newWorldFile.exists()) {
             Logging.fine("Succeeded at copying files");
-            if (this.addWorld(newName, environment, seedString, worldType, generateStructures, generator, useSpawnAdjust)) {
-                // getMVWorld() doesn't actually return an MVWorld
-                Logging.fine("Succeeded at importing world");
-                MVWorld newWorld = (MVWorld) this.getMVWorld(newName);
-                newWorld.copyValues(this.worldsFromTheConfig.get(oldName));
-                // don't keep the alias the same -- that would be useless
-                newWorld.setAlias("");
-                return true;
+
+            // initialize new properties with old ones
+            WorldProperties newProps = new WorldProperties();
+            newProps.copyValues(worldsFromTheConfig.get(oldName));
+            // don't keep the alias the same -- that would be useless
+            newProps.setAlias("");
+            // store the new properties in worlds config map
+            worldsFromTheConfig.put(newName, newProps);
```

Comment:
This is also a really small nitpick, but when referring to `worldsFromTheConfig`, please use `this.worldsFromTheConfig` as it makes it easier to see that it's an attribute of the class.